crates/optimism/cli/src/chainspec.rs

```diff
@@ -21,6 +20,27 @@ fn chain_value_parser(s: &str) -> eyre::Result<Arc<OpChainSpec>, eyre::Error> {
     })
 }
 
+/// Parses a custom [`ChainSpec`].
+pub fn parse_custom_chain_spec(s: &str) -> eyre::Result<ChainSpec, eyre::Error> {
```

Comment:
Moved out of the now deleted `utils.rs` as the only usage appeared to be here in the OpChainSpecParser